### ISSUE TITLE
Questions about apply cmvn

### ISSUE THREAD

**Issue by oshindow** [espnet/espnet#3186]
Hi, I don't understand why the cmvn.ark of `train set` is applied to both `test & dev set` in espnet. It seems that in Kaldi, each applies its own cmvn.ark (e.g., the `test set` will also calculate its own cmvn.ark and apply it). Is this approach because we use e2e model training?

If I replace the test data (from another data set or my own recording), should I also use the cmvn.ark of the `original train set`?

**Comment by sw005320** [comment]
Thanks!
This is a great point.
This cmvn process is intended for the training data whitening purpose, and not intended for the test data normalization purpose (per test set, speaker, or utterance).

You may have some gains if you apply it to the test data, and it would be great if you share your experience about it.

**Comment by oshindow** [comment]
> Thanks!
> This is a great point.
> This cmvn process is intended for the training data whitening purpose, and not intended for the test data normalization purpose (per test set, speaker, or utterance).
> 
> You may have some gains if you apply it to the test data, and it would be great if you share your experience about it.

Thanks for your reply @sw005320 ! I plan to do some experiments on this.
